packages/clay-drop-down/src/DropDown.tsx

```diff
@@ -122,6 +123,16 @@ export interface IProps<T>
 	 */
 	renderMenuOnClick?: boolean;
 
+	/**
+	 * Flag indicating if the caret icon should be displayed on the right side.
+	 */
+	showDropdownIcon?: boolean;
```

Comment:
I think this could be something like `triggerIcon` and maybe we can use that instead of symbol and the undefined value would be used to not render the trigger icon so we can avoid having to have two APIs for this and it makes it a little more intuitive by the API name.

Reply:
Thanks @matuzalemsteles!
I had to make a few changes because of the `DropdownWithItems`.